docs/_includes/date-picker.html

```diff
@@ -0,0 +1,212 @@
+<div id="date-picker">
+    {% include section-header.html name="date-picker" version=page.versions.date-picker %}
+
+    <p>The date picker component is used to display a calendar from which dates can be selected, either as single items or as a range.</p>
+
+    <h3>Single Pane</h3>
+
+    <button class="btn" onclick="document.getElementById('date-picker-example').toggleAttribute('hidden')">Show/Hide</button>
+
+    <div id="date-picker-example" class="date-picker date-picker--single-month" role="dialog" aria-label="date picker">
+        <div class="date-picker__header">
+            <button
+                class="icon-btn icon-btn--small icon-btn--transparent"
+                aria-label="Show July 2019"
+                variant="icon"
+            >
+                <svg aria-hidden="true" class="icon icon--chevron-left" focusable="false">
+                    {% include symbol.html name="chevron-left" %}
+                </svg>
+            </button>
+            <h3>August 2019</h3>
+            <button
+                class="icon-btn icon-btn--small icon-btn--transparent"
+                aria-label="Show July 2019"
```

Comment:
Label should be "Next Month - September 2019". Make sure to change it in the example code also.

Reply:
Yeah, I'll demo it outside a button. I don't know if `date-picker-button` would really be used though because this should ideally be shown as an alternative to typing into a date `input`

Reply:
Yeah, but that alternative is triggered via a postfix icon button in our textbox module:

<img width="655" alt="Screen Shot 2022-12-27 at 11 55 56 AM" src="https://user-images.githubusercontent.com/38065/209716348-652662a2-ed96-4e74-936c-feeb9d946d60.png">

<img width="1123" alt="Screen Shot 2022-12-27 at 11 57 38 AM" src="https://user-images.githubusercontent.com/38065/209716463-12b07b7c-cb39-4092-b741-7d2d22e289e5.png">

Another option could be to just change your example button to be an icon button with calendar.

Reply:
Also, we do want to verify that textbox + icon button + date picker all works together nicely in terms of styling before proceeding to the coreui pieces.